python/requirements.in

```diff
@@ -9,7 +8,7 @@ dbt-postgres
 diagrams>=0.23.3
 faker
 folium
-GDAL==3.8.3
+GDAL==3.9.2
```

Comment:
intentional GDAL version bump? (linking to relevant issue https://github.com/NYCPlanning/data-engineering/issues/843 so this PR shows up in that issue)

Reply:
Oh jeez good catch. Not intentional, will patch now